### ISSUE TITLE
Explorer Folder Does Not Open

### ISSUE THREAD

**Issue by PMoransais** [microsoft/vscode#92266]
Issue Type: <b>Bug</b>

Angular 9 Ivy with new error reporting

After build in the error list when you Control-Click the path to the offending file the file opens correctly but in the explorer the project root folder closes and cannot be reopened.

When the editor for the offending file is closed the explorer expends again

VS Code version: Code 1.42.1 (c47d83b293181d9be64f27ff093689e8e7aed054, 2020-02-11T14:45:59.656Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4790 CPU @ 3.60GHz (8 x 3592)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.93GB (10.19GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (15)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.1.1
gitlens|eam|10.2.1
EditorConfig|Edi|0.14.4
vscode-npm-script|eg2|0.3.11
vscode-firefox-debug|fir|2.7.0
azure-storage-explorer|for|0.1.2
vscode-azureappservice|ms-|0.16.3
vscode-cosmosdb|ms-|0.12.1
azure-account|ms-|0.8.8
team|ms-|1.161.0
debugger-for-chrome|msj|4.12.6
debugger-for-edge|msj|1.0.14
vscode-css-peek|pra|3.0.2
code-spell-checker|str|1.8.0
vscodeintellicode|Vis|1.2.6

</details>
We have written the needed data into your clipboard because it was too large to send. Please paste.

**Comment by isidorn** [comment]
Thanks for filling this issue. Can you reproduce using Visual Studio Code Insiders https://code.visualstudio.com/insiders/

**Comment by PMoransais** [comment]
The issue does not happen with the insider version. 

I do not know if it matters but I was running terminal in Legacy mode in the latest version of VS. When I started the Insider version the terminal window displayed outside of vs code as a standard powershell window with no active link to the files with errors. I removed legacy mode from the Powershell window (under properties) and the terminal windows displayed inside vs, the links worked and the explorer updated properly. However, canceling legacy mode did not change the behavior of the current version of vs

I hope it helps

Thanks for the quick response
